### PR TITLE
Add/layer ordering

### PR DESCRIPTION
**Pull request by matt-in-a-hat**
Not sure why I didn't create a PR for this 5 months ago...

Pulled dependencies into package.json
Update to respect sort order config for layers

### CODE REVIEW

File: wrappers/map.js [hunk 1]
```diff
@@ -117,20 +119,42 @@ class Map extends EventEmitter {
...
+      }
+    })
+    sortableLayers.sort(function (a, b) {
+      return a.options.sortOrder - b.options.sortOrder
+    })
+    sortableLayers.forEach(function (layer) {
+      try {
+        layer.bringToFront()
+      } catch (e) {}
+    })
```
Comment (markstuart) [inline wrappers/map.js]:
```js
this.geojsonlayers
  .map(...)
  .sort(...)
  .forEach(...)
```

Comment (markstuart) [inline wrappers/map.js]: Actually, the map is really a filter

Comment (matt-in-a-hat) [inline wrappers/map.js]: Does it work on an object?

Comment (markstuart) [inline wrappers/map.js]: Oh, sorry, forgot about that. Where is `R.values` when you need it?? What about `Object.keys(obj).map(key => obj[key]).filter().sort().etc` ?

**Pull request merged**